DEVELOPMENT.md

```diff
@@ -18,7 +18,7 @@ On Ubuntu (or on Windows [using the Windows Subsystem for Linux](https://www.mic
 curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
 sudo apt-get install -y build-essential nodejs
 
-curl -sSL https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz | sudo tar -xz -C /usr/local
+curl -sSL https://dl.google.com/go/go1.12.3.linux-amd64.tar.gz | sudo tar -xz -C /usr/local
```

Comment:
I guess this would not be `1.12.4`.. 👍

Reply:
Not really needed. 1.12.4 fixes an issue related to cgo, which we don't use.